octane_api_credits.go

```diff
@@ -20,7 +20,7 @@ type (
 		// Optional description. This is only viewable internally
 		Description string `json:"description,omitempty"`
 		// Total price paid for the credits in cents. Defaults to $1 (100 cents) per credit if not specified
-		Price int32 `json:"price,omitempty"`
+		Price int32 `json:"price"`
```

Comment:
Something doesn't seem right in this change, FWIW. Specifically, the comment doesn't match the behavior, because now if you don't set this explicitly, it will get sent as `0`, which I imagine will set it as a price of 0.

Given this is an optional field, it feels like it should be an `*int32` so the caller can distinguish "not set" vs 0 price.